File: test_download.py
import urllib.request
req = urllib.request.Request('https://nexaris-750648121075.europe-west1.run.app/api/v1/download/apk', method='GET')
try:
    with urllib.request.urlopen(req) as response:
        print(response.getcode())
        print(response.headers)
        data = response.read(1024)
        print('First bytes:', data[:4])
except Exception as e:
    print(e)

File: test_download.js
const https = require('https');
https.get('https://nexaris-750648121075.europe-west1.run.app/api/v1/download/apk', (res) => {
  console.log('Status Code:', res.statusCode);
  console.log('Headers:', res.headers);
  res.on('data', (d) => {
    console.log('First chunk length:', d.length);
    console.log('First 4 bytes:', d.toString('hex', 0, 4));
    process.exit(0);
  });
}).on('error', (e) => {
  console.error(e);
});
